apps/web/screens/PetitionView/PetitionView.tsx

```diff
@@ -253,4 +258,31 @@ const PetitionView = ({ namespace }: PetitionViewProps) => {
   )
 }
 
+interface InitialProps {
```

Comment:
You could type the component as a Screen instead of creating this InitialProps interface

Reply:
I commited that change with your permission :)